src/core/lib/security/authorization/rbac_policy.h

```diff
@@ -162,15 +172,22 @@ struct Rbac {
   };
 
   Rbac() = default;
-  Rbac(Rbac::Action action, std::map<std::string, Policy> policies);
+  Rbac(Rbac::Action action, std::map<std::string, Policy> policies,
+       absl::string_view name);
 
   Rbac(Rbac&& other) noexcept;
   Rbac& operator=(Rbac&& other) noexcept;
 
   std::string ToString() const;
 
   Action action;
+  AuditCondition audit_condition;
```

Comment:
Please move this down next to the `logger_configs` field, since they're closely related.